workflows/sock-shop-demo/usingPrompPobe/workflow.yaml

```diff
@@ -24,6 +24,8 @@ spec:
             template: install-pod-memory-hog
           - name: install-pod-cpu-hog
             template: install-pod-cpu-hog 
+          - name: install-pod-network-loss
```

Comment:
Can you check the name of this file? Do you mean usingPromProbe? check for other places also